src/report-portal.js

```diff
@@ -201,6 +201,11 @@ class ReportPortal {
      */
 
     async _startStep(time, name = "'->") {
+
+        // Adding gard of the lentgh of the group name 
+        if (name.length > 120) {
```

Comment:
We should keep an eye on illegal chars, etc on the future , more input guards might be needed here